Modules/VMware.Hv.Helper/VMware.HV.Helper.psm1

```diff
@@ -10001,10 +10001,10 @@ function Set-HVApplicationIcon {
             Write-Host "Some application(s) already have an association for the specified icon."
             $ApplicationIconHelper.ApplicationIcon_UpdateAssociations($services, $applicationIconId, @($appInfo.Id))
             Write-Host "Successfully updated customized icon association for Application:[$ApplicationName]."
-            break
-          }
-          Write-Host "Error in associating customized icon for Application:[$ApplicationName] $_"
-          break
+          } else {
+          	Write-Host "Error in associating customized icon for Application:[$ApplicationName] $_"
+          	break
```

Comment:
Remove this break and move the success message that's currently after it in the try block. Currently in the EntityAlreadyExists situation you'll get two identical success messages one after the other.

Reply:
@kamennikolov I have remove the unnecessary Write-Host